src/cron/bundle.rs

```diff
@@ -276,18 +276,21 @@ fn verify_tx_receipt(tx_receipt: &TxReceipt) -> std::io::Result<bool> {
     ]))
     .unwrap();
 
+    // TODO: remove this lazy static and use KeyManager from context
     lazy_static! {
         static ref PUBLIC: PKey<Public> = {
-            let jwk = JWK {
-                kty: "RSA",
-                e: "AQAB",
-                n: std::env::var("BUNDLER_PUBLIC").unwrap(),
-            };
-
-            let p = serde_json::to_string(&jwk).unwrap();
-            let key: JsonWebKey = p.parse().unwrap();
+            let jwk = JsonWebKey::new(Key::RSA {
+                public: RsaPublic {
+                    e: PublicExponent,
+                    n: BASE64URL_NOPAD
```

Comment:
Shouldn't this be just `std::env::var("BUNDLER_PUBLIC").unwrap()`?

This string already comes Base64 encoded

Reply:
Yes, but we need the decoded string. I've pushed a test case that shows this.